environment.conf.json

```diff
@@ -54,6 +54,7 @@
       "gemeentenaam": "amsterdam"
     }
   },
+  "hasExtraProps": true,
```

Comment:
I think the name of this prop is too generic. Should be something like `showVulaanControls`. At least use naming that has already been used in the application.

Reply:
shure thing!